### PR TITLE
CoinMarketCap presence

### PR DESCRIPTION
**Pull request by jonasrdl**
I hope everything fits, if not I'll try to fix it.

### CODE REVIEW

**Comment by EncryptedDev** [comment]
Upload proof

**Comment by jonasrdl** [comment]
> Upload proof

how many screenshots do you need?  I think it makes no sense to screenshot all 30 variants.

**Comment by EncryptedDev** [comment]
> > Upload proof
> 
> how many screenshots do you need? I think it makes no sense to screenshot all 30 variants.

Two or three should be sufficient

**Comment by jonasrdl** [comment]
> > > Upload proof
> > 
> > 
> > how many screenshots do you need? I think it makes no sense to screenshot all 30 variants.
> 
> Two or three should be sufficient

image

image

image

image

File: websites/C/CoinMarketCap/presence.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  else if (path.includes("/faq/")) presenceData.details = "Reading the FAQ";
+  else if (path.includes("/alexandria/glossary"))
+    presenceData.details = "Reading the glossary";
+  else if (path.includes("/trending-cryptocurrencies/"))
+    presenceData.details = "Browsing the trending cryptocurrencies";
+
+  if (!presenceData.details) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else presence.setActivity(presenceData);
```
Comment (Slowlife01) [inline websites/C/CoinMarketCap/presence.ts:98]:
```suggestion
  if (!presenceData.details) presence.setActivity();
  else presence.setActivity(presenceData);
```

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**